### ISSUE TITLE
not working!!! just can find out where is the face in debug

### ISSUE THREAD

**Issue by johncheung233** [lafosca/AndroidFaceCropper#9]

**Comment by 2shrestha22** [comment]
Yes exactly. However his app seems working. Do you found any solution?

**Comment by 2shrestha22** [comment]
It was not working because our image size is less than minFaceSize = 200. You need to change the minFaceSize to get it worked. I did
`mFaceCropper.setFaceMinSize(20);`
and it worked.

**Comment by johncheung233** [comment]
thanks your help




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Sangam Kumar Shrestha"<notifications@github.com&gt;; 
发送时间: 2019年11月26日(星期二) 上午10:14
收件人: "lafosca/AndroidFaceCropper"<AndroidFaceCropper@noreply.github.com&gt;; 
抄送: "John Cheung"<875677993@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [lafosca/AndroidFaceCropper] not working!!! just can find out where is the face in debug (#9)




It was not working because our image size is less that minFaceSize = 20. You need to change the minFaceSize to get it worked. I did
 mFaceCropper.setFaceMinSize(20);
 and it worked.

**Comment by johncheung233** [comment]
I try,but fail.Finally I abandon the cut function and just keep the function which can find out the face. My english is not good LOL




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Sangam Kumar Shrestha"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月26日(星期二) 上午10:14
收件人:&nbsp;"lafosca/AndroidFaceCropper"<AndroidFaceCropper@noreply.github.com&gt;;
抄送:&nbsp;"John Cheung"<875677993@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [lafosca/AndroidFaceCropper] not working!!! just can find out where is the face in debug (#9)




It was not working because our image size is less that minFaceSize = 20. You need to change the minFaceSize to get it worked. I did
 mFaceCropper.setFaceMinSize(20);
 and it worked.
